opentelemetry/src/main/scala/zio/telemetry/opentelemetry/ContextPropagation.scala

```diff
@@ -48,11 +46,6 @@ private[opentelemetry] object ContextPropagation {
     carrier: C,
     setter: HttpTextFormat.Setter[C]
   ): URIO[Tracing, Unit] =
-    ZIO.uninterruptible {
-      for {
-        context <- UIO(TracingContextUtils.withSpan(span, Context.ROOT))
-        _       <- UIO(httpTextFormat.inject(context, carrier, setter))
-      } yield ()
-    }
+    UIO(TracingContextUtils.withSpan(span, Context.ROOT)).map(httpTextFormat.inject(_, carrier, setter))
```

Comment:
This one needs not be uninterruptible AFAIKS, as the inject uses the return value of `withSpan` instead of reading a ThreadLocal.

Reply:
Responded with https://github.com/zio/zio-telemetry/pull/119#discussion_r412207764